src/platform/datapath_xplat.c

```diff
@@ -137,7 +137,8 @@ CxPlatSocketCreateUdp(
         SocketCreateUdp(
             Datapath,
             Config,
-            NewSocket);
+            NewSocket
+        );
```

Comment:
```suggestion
            NewSocket);
```
Please don't change style.